include/deal.II/matrix_free/tools.h

```diff
@@ -1054,16 +1543,165 @@ namespace MatrixFreeTools
             for (unsigned int i = 0; i < phi.dofs_per_cell; ++i)
               {
                 helper.prepare_basis_vector(i);
-                local_vmult(phi);
+                cell_operation(phi);
                 helper.submit();
               }
 
             helper.distribute_local_to_global(diagonal_global_components);
           }
-      },
-      diagonal_global,
-      dummy,
-      false);
+      };
+
+    const auto face_operation_wrapped =
+      [&](const MatrixFree<dim, Number, VectorizedArrayType> &matrix_free,
+          VectorType &,
+          const int &,
+          const std::pair<unsigned int, unsigned int> &range) mutable {
```

Comment:
I know we already had this `mutable` declaration in the old code a few lines above, but could you quickly explain why it is needed in this place? I do not see any of the state be modified that would be in a `const` context, but I probably just miss that part.

Reply:
It seems we can remove the `mutable`.